.horde.yml

```diff
@@ -30,36 +30,36 @@ authors:
     active: false
     role: lead
 version:
-  release: 4.0.0
-  api: 4.0.0
+  release: 4.0.0alpha7
+  api: 4.0.0alpha1
 state:
-  release: beta
-  api: beta
+  release: alpha
+  api: alpha
 license:
   identifier: ASL
   uri: http://www.horde.org/licenses/apache
 dependencies:
   required:
-    php: ^5.3 || ^7
-    pear:
-      pear.horde.org/horde: ^5
```

Comment:
@boekhorstb1 is your branch maybe not rebased correctly? For example I don't see these entries in the target branch: https://github.com/maintaina-com/ingo/blob/FRAMEWORK_6_0/.horde.yml